ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/SourceDescription.java

```diff
@@ -78,22 +82,39 @@ public SourceDescription(
     this.timestamp = timestamp;
     this.statistics = statistics;
     this.errorStats = errorStats;
+    this.metrics = metrics;
```

Comment:
I know this class doesn't for its other fields - but we should think about now allowing nulls - either reject them or default to safe value.